charts/stable/datapower-operator/Chart.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v2
+name: datapower-operator
+description: A chart to deploy the DataPowerService Operator
+type: application
+version: 1.0.0
+appVersion: 0.11.0
```

Comment:
@dmnewber Should this be changed?

Reply:
It sure should. Won't affect anything functional, but it is wrong.